apps/www/src/lib/registry/new-york/ui/chart-area/AreaChart.vue

```diff
@@ -83,7 +83,7 @@ function handleLegendItemClick(d: BulletLegendItemInterface, i: number) {
 
       <template v-for="(category, i) in categories" :key="category">
         <VisArea
-          :x="(d: Data, i: number) => i"
+          :x="(_: Data, i: number) => i"
```

Comment:
perfer `_d` instead of `_`

```suggestion
          :x="(_d: Data, i: number) => i"
```

Reply:
> perfer `_d` instead of `_`

Done.